### PR TITLE
New-style errors: Nicer tooltip

### PR DESCRIPTION
**Pull request by janezd**

### CODE REVIEW

File: Orange/widgets/utils/messages.py [hunk 1]
```diff
@@ -325,16 +325,16 @@ def update_message_state(self):
...
             # TODO: fix tooltip background color - it is not white
-            tooltip = '<div style="background-color: #ffffff;">{}</div>'.format(
-                ''.join(
-                    '<p style="background-color: {}; padding: 5">'
-                    '<nobr>{}</nobr></p>'.format(
-                        msg.group.bar_background, str(msg))
-                    for msg in messages))
+            tooltip = ''.join(
+                '<p style="background-color: {}; margin: 0;">'
+                '<nobr style="font-size: {}px;">&nbsp;{}&nbsp;</nobr></p>'.
```
Comment (kernc) [inline Orange/widgets/utils/messages.py]: Font sizing doesn't work or is too small. Some padding would be nice.

screenshot - 07292016 - 12 32 49 pm

Comment (kernc) [inline Orange/widgets/utils/messages.py]: This adds vertical padding for me.
```html
<p style="background-color: {}; margin: 0;">
<span style="font-size:9pt"><br></span>
<nobr style="font-size: {}px;">&nbsp;&nbsp;&nbsp;
{}
&nbsp;&nbsp;&nbsp;</nobr>
<span style="font-size:9pt"><br></span>
</p>

**Comment by ajdapretnar** [comment]
@janezd Can the padding be fixed so we can merge? Tnx. :star:

**Comment by janezd** [comment]
Done.

**Pull request merged**